Removed condescending language from Command Reference (#1522)
iterative/dvc.org: content/docs/command-reference/fetch.md

## Examples

Let's employ a <abbr>workspace</abbr> with some data, code, ML models, pipeline

> stages, such as the <abbr>DVC project</abbr> created for the

Reviewer: Same here: new workspace; fresh workspace; clean workspace — something like that

    deleted:            data/data.xml
```

> One could do a `dvc fetch` to get all the data, but what if you only want to

Reviewer: ```suggestion
We could use `dvc fetch` to get all the data, but what if you only want to
```
